<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inventarioWindow</class>
 <widget class="QMainWindow" name="inventarioWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="inventarioLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INVENTARIO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nuevo_item">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(15, 67, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>NUEVO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonModificarStock">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 249, 69);
color: rgb(40, 40, 40)</string>
    </property>
    <property name="text">
     <string>STOCK</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetInventario">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>311</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Material</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cantidad</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="irAlMenu">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>300</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ir al menu Principal</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>557</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
